tests/component/test_export_signal_properties.py

```diff
@@ -0,0 +1,63 @@
+"""Contains a collection of pytest tests that validates export signal properties."""
+
+import pytest
+
+from nidaqmx.constants import ExportAction
+from nidaqmx.error_codes import DAQmxErrors
+from nidaqmx.errors import DaqError
+from nidaqmx.task import Task
+
+
+def test__export_signal__get_enum_property__returns_value(ai_voltage_chan_task: Task):
+    """Test to validate getter for export signal property of enum type."""
+    assert ai_voltage_chan_task.export_signals.samp_clk_output_behavior == ExportAction.PULSE
+
+
+def test__export_signal__set_enum_property__returns_assigned_value(ai_voltage_chan_task: Task):
+    """Test to validate setter for export signal property of enum type."""
+    value_to_set = ExportAction.LEVEL
+    ai_voltage_chan_task.export_signals.samp_clk_output_behavior = value_to_set
+
+    assert ai_voltage_chan_task.export_signals.samp_clk_output_behavior == value_to_set
+
+
+def test__export_signal__reset_enum_property__returns_initial_value(ai_voltage_chan_task: Task):
+    """Test to validate resetting export signal property of enum type."""
+    ai_voltage_chan_task.export_signals.samp_clk_output_behavior == ExportAction.INTERLOCKED
+
+    del ai_voltage_chan_task.export_signals.samp_clk_output_behavior
+
+    assert ai_voltage_chan_task.export_signals.samp_clk_output_behavior == ExportAction.PULSE
+
+
+def test__export_signal__get_string_property__returns_value(ai_voltage_chan_task: Task):
+    """Test to validate getter for export signal property of string type."""
+    assert ai_voltage_chan_task.export_signals.start_trig_output_term == ""
+
+
+def test__export_signal__set_invalid_routing_destination__throws_daqerror(
+    ai_voltage_chan_task: Task,
+):
+    """Test to validate setter for export signal property of string type."""
+    ai_voltage_chan_task.export_signals.start_trig_output_term = "RTSI"
+
+    with pytest.raises(DaqError) as e:
+        _ = ai_voltage_chan_task.export_signals.start_trig_output_term
+    assert e.value.error_type == DAQmxErrors.INVALID_ROUTING_DESTINATION_TERMINAL_NAME_ROUTING
+
+
+def test__export_signal__set_string_property__returns_assigned_value(ao_voltage_chan_task: Task):
+    """Test to validate setter for export signal property of string type."""
+    value_to_set = "RSE"
+    ao_voltage_chan_task.export_signals.start_trig_output_term = value_to_set
+
+    assert ao_voltage_chan_task.export_signals.start_trig_output_term == value_to_set
+
+
+def test__export_signal__reset_string_property__returns_initial_value(ao_voltage_chan_task: Task):
```

Comment:
Change all references to "initial_value" to "default_value".

Reason: "default value" is the standard DAQmx terminology